invenio_audit_logs/services/generators.py

```diff
@@ -19,6 +19,6 @@ class CurrentUser(Generator):
 
     def needs(self, identity=None, **kwargs):
         """Check if the user identity matches the identity in the request."""
-        if identity:
+        if current_user.is_authenticated:
```

Comment:
This looks strange. `current_user` can also be anonymous.
Here, when not authenticated, we return `system process`. Is this wanted?

Reply:
to be removed

Reply:
still to be addressed

Reply:
removed.